sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -626,6 +626,30 @@ public virtual Pageable<ConfigurationSetting> GetConfigurationSettings(SettingSe
             return PageResponseEnumerator.CreateEnumerable(nextLink => GetConfigurationSettingsPage(selector, nextLink, cancellationToken));
         }
 
+        /// <summary>
+        /// Retrieves the different revisions of specific <see cref="ConfigurationSetting"/> with specified <paramref name="key"/> and <paramref name="label"/>.
```

Comment:
"of a specific"
"with the specified"

I realize this was copied from elsewhere - would you mind updating those instances as well?

Reply:
Fixed